streaming-server/nginx.conf

```diff
@@ -17,21 +19,20 @@ rtmp {
 
             on_publish http://auth_server:8000/auth;
             # TODO: turn the publish endpoint dynamic like rtmp://localhost:1935/hls-live/<anyname>
-            push rtmp://localhost:1935/hls-live;
-
+            # push rtmp://localhost:1935/hls-live;
         }
 
-    # TODO: take the params and set as variable
         application hls-live {
             live on;
             record off;
-            allow 127.0.0.1;
-            deny all;
-
+            # allow 127.0.0.1;
```

Comment:
By removing this you are not exposing the `hls-live` without passing by the auth server?

Reply:
In this [commit](https://github.com/lalizita/streaming-key-server-manager/pull/7/commits/e578e96251639918013f7bb2e7b9c15074a20fe7) I've readded the restriction to allow publish to `hls-live` only for `127.0.0.1`